tests/common.go

```diff
@@ -114,18 +150,18 @@ func runACBuild(workingDir string, args ...string) *acbuildError {
 	err = cmd.Wait()
 	if exitErr, ok := err.(*exec.ExitError); ok {
 		code := exitErr.Sys().(syscall.WaitStatus).ExitStatus()
-		return &acbuildError{exitErr, code, stdout, stderr}
+		return code, stdout, stderr, fmt.Errorf("non-zero exit code of %d: %v\nstdout:\n%s\nstderr:\n%s", code, exitErr, string(stdout), string(stderr))
```

Comment:
how about just override the stdout, stderr to a bytes.Buffer?
e.g. https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/rkt/rkt.go#L200